docs/hugo/content/guide/authentication/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Authentication in Azure Service Operator"
+linkTitle: "Authentication"
+layout: single
+cascade:
+- type: docs
+- render: always
+---
+
+There are two key topics surrounding authentication in Azure Service Operator: The type of credential, and the credential scope.
+
+## Credential type
+
+Azure Service Operator supports four different styles of authentication today.
+
+1. [Service Principal using a Client Secret](./credential-format.md#service-principal-using-a-client-secret)
+2. [Service Principal using a Client Certificate](./credential-format.md#service-principal-using-a-client-certificate)
+3. [Azure-Workload-Identity](https://github.com/Azure/azure-workload-identity) authentication (OIDC + Managed Identity or Service Principal)
+4. [Deprecated] aad-pod-identity authentication (Managed Identity)
+
+## Credential scope
+
+Each supported credential type can be specified at one of three supported scopes:
+
+1. [Global](./credential-scope.md#global-scope) - The credential applies to all resources managed by ASO.
+2. [Namespace](./credential-scope.md#namespace-scope) - The credential applies to all resources managed by ASO in that namespace.
+3. [Resource](./credential-scope.md#resource-scope) - The credential applies to only the specific resource it is referenced on.
+
+When presented with multiple credential choices, the operator chooses the most specific one:
+per-resource takes precedence over per-namespace which takes precedence over global.
```

Comment:
```suggestion
_per-resource_ takes precedence over _per-namespace_ which takes precedence over _global_.
```

Reply:
I changed to use `resource scope` rather than `per-resource` as I think that reads better. I did add the emphasis you suggested here though